packages/sentry/src/sentry.ts

```diff
@@ -39,15 +42,15 @@ const replaceValues = <T>(source: T, replacements: TReplacements): T => {
       switch (typeof source[prop]) {
```

Comment:
Removing the generic caused `No index signature with a parameter of type 'string' was found on type 'Event'.` which I don't fully understand as event has string as keys.